### PR TITLE
Errors

### PR DESCRIPTION
**Pull request by rexielnino**

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -12,6 +12,8 @@
   // rate of failing to fire torpedos [0.0, 1.0]
   private double FAILURE_RATE = 0.0; //NOSONAR
 
+  Random generator = new Random();
```
Comment (kovacsakos) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: Make field private.

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -30,18 +32,18 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```
Comment (rexielnino) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:35]: gyönyörű munka!

**Review by kovacsakos — CHANGES_REQUESTED** [review]
Make `generator` field private before merge.

**Review by rexielnino — COMMENTED** [review]
-

**Comment by rexielnino** [comment]
Changed

**Review by kovacsakos — APPROVED** [review]

**Pull request merged**